predict_loading_concentration.py

```diff
@@ -4,35 +4,39 @@ def warn(*args, **kwargs):
 import warnings
```

Comment:
This file should be deleted altogether. It is from some previous version (you see a different package name `diva_dna_seq`). The one that is used is diva_seq_opt/predict_loading_concentration.py.

Reply:
This was much more than that...
The predict_loading_concentration() was in the __init__.py file, and the main predict_loading_concentration.py seems more recent at the parent directory level. See most recent commit.

There is still room for improvement, but at least now we have only one predict_loading_concentration.py, and the predict_loading_concentration() is in the utility.py, not in __init__.py.

Reply:
Good progress. But now there will be a problem in setup.py file, which used to work with `predict_loading_concentration.py` file that was in `diva_seq_opt` dir. See line 11 in setup.py. 
What I would do is change to
``
entry_points = {        
'console_scripts': ['predict-loading-concentration=predict_loading_concentration:main'],      
},
``
This should be tested after a fresh install. 
Note the change `_`->`-` in the command line tool. I would do that to distinguish from the module and function with the same name, but I leave it to you.

Reply:
For now I think keeping the underscores might be better for the sole purpose of more shell support. I just read that some shells don't allow dashes in function names:
https://unix.stackexchange.com/questions/482417/will-it-be-bad-that-a-function-or-script-name-contains-dash-instead-of-under